### PR TITLE
Logged-Out Theme Showcase: Update SEO Content

### LINKED ISSUE #78225: Theme Showcase: Show different header and description text when logged out
## Description

From https://github.com/Automattic/dotcom-forge/issues/2410:

> On the logged out /themes page, we would like to provide better SEO content and additional content for category page views. And explore design ideas for making the category area on the /themes page more attractive. We can break this into two parts.

This issue will focus on updating header and description texts for different categories, plus a title tag fix:
Image

### Specs

Navigate to the logged-out Theme Showcase (`https://wordpress.com/themes/`) and update the page's copy as specified by the [spreadsheet](https://docs.google.com/spreadsheets/d/10uoJqOZLTne3565nbbR6ggR09vrPhUpK_ztwbuvau14/edit#gid=0).

Screenshot 2023-06-27 at 15 17 54

#### Changes

- `<title>` tag
- `<meta name="description">` tag
- Header
- Description

Remember to consider the translations before deploying (see the team's handbook: PaYJgx-3sX-p2).

## Dev Tasks
- [ ] Update the copy as specified in the [spreadsheet](https://docs.google.com/spreadsheets/d/10uoJqOZLTne3565nbbR6ggR09vrPhUpK_ztwbuvau14/edit#gid=0)
- [ ] Update the title on https://wordpress.com/themes/marketplace to be `WordPress Marketplace Themes` as discussed [here](pdyEEc-st-p2#comment-792) and [here](https://docs.google.com/spreadsheets/d/10uoJqOZLTne3565nbbR6ggR09vrPhUpK_ztwbuvau14/edit#gid=0&range=F13) 
- [ ] Ask @sailerbe  to review when ready

### PR DESCRIPTION
**Pull request by Copons**
Fixes https://github.com/Automattic/wp-calypso/issues/78225

## Proposed Changes

* Update title, meta description, header and sub-header of the Logged-Out Theme Showcase so that different filters will result in different copy.

See the [source spreadsheet](https://docs.google.com/spreadsheets/d/10uoJqOZLTne3565nbbR6ggR09vrPhUpK_ztwbuvau14/edit#gid=0) for the proposed copy.

| Before | After |
|--------|--------|
| Screenshot 2023-07-25 at 18 41 08 | Screenshot 2023-07-25 at 18 41 13 | 

## Testing Instructions

* While logged-out, visit `/themes`.
  * If you are running Calypso locally, ensure [the SSR is in debug mode](https://github.com/Automattic/wp-calypso/blob/trunk/docs/server-side-rendering.md#debugging).
* Change subjects and pricing filters.
* Check the page's heading and sub-heading, and ensure the copy matches the hardcoded list provided in this PR.
* Inspect the page source, and ensure the `<title>` and `<meta name="description">` tags too match the hardcoded list.
* Open `/themes` while logged-in.
* Ensure the header looks like this and does not changes depending on the filters.
  * Screenshot 2023-07-25 at 18 48 45

### CODE REVIEW

**Comment by Copons** [comment]
@sailerbe There are a few differences compared to the spreadsheet:

- Added copy for the `about` category, which was missing from the spreadsheet.
- I have not added copy for `single-page` and `photography` as not part of the category list.
- I also have not added copy for `podcast` which, though, should become part of the list soon-ish.
  - When the new categories are ready, we'll have to revisit this list anyway to add copy for any missing category.

File: client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js
Comment (mmtr) [inline client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js]: This doesn't match the spreadsheet
```suggestion
			header: translate( 'Find the perfect theme for your link in bio website.' ),
```

File: client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js
Comment (mmtr) [inline client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js]: This indeed matches the spreadsheet, but I think the spreadsheet is wrong and this should be like this:

```suggestion
				"Enhance your business website with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
```

File: client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js
Comment (mmtr) [inline client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js]: This causes the title / header / description to suddenly change when filtering by feature.

See this example and note how the header changes from "Find the perfect theme for your blog." to "Find the perfect theme for your website." after selecting a feature:


https://github.com/Automattic/wp-calypso/assets/1233880/a7e78968-fda4-401e-a9ea-ba3379509347




I think it should remain as "Find the perfect theme for your blog." since the Blog category is still selected.

Comment (Copons) [reply client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js]: Excellent catch @mmtr! Should be fixed with f2c6f3f.

File: client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js
Comment (mmtr) [inline client/my-sites/themes/get-theme-showcase-logged-out-seo-content.js]: Still wrong I think!
```suggestion
				"Enhance your business website with the perfect premium theme. Available on all Premium, Business, and Ecommerce plans. Activate the one that's right for you."
```

**Review by mmtr — APPROVED** [review]
Looks good! Do we need to add the "String freeze" label before merging?

**Comment by Copons** [comment]
> Looks good! Do we need to add the "String freeze" label before merging?

Yes! I'm waiting for @sailerbe to give this a last check before sending the whole thing to translation. 🙂

**Comment by sailerbe** [comment]
@Copons What's the best way to preview the changes (still learning my way around this process)?

**Comment by Copons** [comment]
> @Copons What's the best way to preview the changes (still learning my way around this process)?

@sailerbe replied in private, but anyway. 🙂 

There's an [automated comment](https://github.com/Automattic/wp-calypso/pull/79845#issuecomment-1650283115) to this PR containing a Calypso Live link.

It builds an instance of Calypso with this PR applied; once built, it redirects to an address like `container-nifty-driscoll.calypso.live`.
Just open it in incognito (to easily see the logged-out experience) and navigate to `/themes`.
E.g. `https://container-nifty-driscoll.calypso.live/themes`

I've confirmed that all the server-side rendered parts are working, so it should only be a matter of checking whether the copy is correct and it looks as expected on the page.

**Comment by mmtr** [comment]
> before sending the whole thing to translation

What if we replace "paid theme(s)" with "partner theme(s)" now in the SEO content so translators don't need to translate again some sentences in https://github.com/Automattic/wp-calypso/pull/79871?

**Comment by Copons** [comment]
> > before sending the whole thing to translation
> 
> What if we replace "paid theme(s)" with "partner theme(s)" now in the SEO content so translators don't need to translate again some sentences in #79871?

I've totally lost track of that conversation, but it seems we have reached an agreement for `Partner` — see paYJgx-3AU-p2#comment-3793 (cc @autumnfjeld).

Then yeah, we could proceed with replacing `paid` with `partner`.

@sailerbe do you have objections from an editorial standpoint?

Some examples to give an idea:

| Paid | Partner |
|--------|--------|
| Browse all paid themes for WordPress.com. Activate the one that's right for you, risk-free for 14 days. | Browse all partner themes for WordPress.com. Activate the one that's right for you, risk-free for 14 days. |
| Improve your link in bio website with the perfect paid theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you. | Improve your link in bio website with the perfect partner theme. Browse high-quality, professionally-designed options. Then, purchase the one that's right for you. |

IMHO, it looks good _in theory_, but something seems a bit off to me. I'm unsure whether I'm biased as a non-native English speaker.

I'll wait for confirmation from Autumn and Ben before updating the copy.

**Comment by sailerbe** [comment]
@Copons No objections.

**Comment by sailerbe** [comment]
@Copons Got it, thanks for the explanation. When opening in an incognito window, the preview link is showing me a blank screen with a Dotcom header banner across the top. Is there something I might be doing wrong?

**Comment by mmtr** [comment]
@sailerbe you're not doing anything wrong – you just need to manually add the `/themes` path to the URL:



https://github.com/Automattic/wp-calypso/assets/1233880/6559965d-56a4-4dac-9b6e-f1c9b9fb36c2

**Comment by Copons** [comment]
> @Copons No objections.

@sailerbe Excellent!
I'll wait for the current product decision over the Paid vs Partner term, and then send for translation whatever is chosen. 🙂

**Comment by sailerbe** [comment]
@Copons The copy updates look great! For the About category pages, could we change "about" to About Me (with this capitalization)? "about" on its own reads a bit odd in this context. 

Everything else looks good. Thanks for getting this done (and thanks @mmtr for the explanation on how to preview the pages correctly).

**Comment by Copons** [comment]
New updates:

- Replaced `About`/`about` with `About Me`.
- Replaced `Paid`/`paid` with `Partner`/`partner`.

**Review by mmtr — APPROVED** [review]

**Review by sailerbe — APPROVED** [review]

**Comment by sailerbe** [comment]
@Copons Looks good to me!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8659400

Thank you @Copons for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8659400).

**Pull request merged**
